### DISCUSSION TITLE
Customize interactors mouse bindings

### DISCUSSION THREAD

**Discussion by toro01** [pyvista/pyvista#4043] [Q&A] [answered]
Hi PyVista team,

Our users come from a software that includes its own 3D scene that can be manipulated with the mouse but is not based on PyVista.

Problem is that the bindings between mouse buttons and actions is different from what is implemented by default in PyVista.

So I would like to customize the bindings between mouse buttons and the scene interactor to made the transition smoother for our users. I had a look at the code, and I think bindings are hard-coded. Do you have any example of how to change that, or does it require any dev? 

Thanks a lot for the good work!

**Comment by jsaintauret** [comment, marked as answer]
Hi, I think I asked a similar question couples weeks ago and managed to find a solution, maybe you should take a look on that thread [https://github.com/pyvista/pyvista/discussions/3823](https://github.com/pyvista/pyvista/discussions/3823)

**Comment by MatthewFlamm** [comment]
The next version of PyVista (0.44) will allow choosing which mouse move action occurs with each mouse button press type for the default style.  This will be `Plotter.enable_custom_trackball_style`.
